INTEGRATE.md

```diff
@@ -0,0 +1,32 @@
+# Local integration between Porta/System with Keycloak through Zync
```

Comment:
I think maybe an opening sentence about what this integration solves would make sense here. 

e.g. with all 3 components installed, you can do X, Y and Z... and maybe the role each plays in that. 

(not long text though - just a tl;dr kind of thing)